misk/src/main/kotlin/misk/web/WebModule.kt

```diff
@@ -17,10 +18,17 @@ import misk.web.interceptors.MetricsInterceptor
 import misk.web.interceptors.PlaintextInterceptorFactory
 import misk.web.interceptors.RequestLoggingInterceptor
 import misk.web.jetty.JettyModule
+import javax.servlet.http.HttpServletRequest
 
 class WebModule : KAbstractModule() {
     override fun configure() {
         install(JettyModule())
+        install(object: ActionScopedProviderModule() {
+            override fun configureProviders() {
+                bindSeedData(Request::class)
+                bindSeedData(HttpServletRequest::class)
```

Comment:
Yeah I think this might be a mistake. Lets discuss?

Reply:
(I trust like 5 people to use this successfully; for everyone else I think it’s too dangerous)

Reply:
If we don't do this, most interceptors can't function - they need access to e.g. certs, headers, etc. We can selectively expose those things onto Request though